ts/screens/profile/PreferencesScreen.tsx

```diff
@@ -182,8 +184,11 @@ class PreferencesScreen extends React.Component<Props, State> {
             <ListItemComponent
               title={I18n.t("profile.preferences.list.email")}
               subTitle={profileData.spid_email}
-              iconName={"io-email"}
-              onPress={unavailableAlert}
+              onPress={() =>
```

Comment:
Since this feature is under development use _**isEmailEditingAndValidationEnabled**_ configuration flag

Reply:
Updated within https://github.com/teamdigitale/io-app/pull/1363/commits/43aa434432e65ac03d1fa9f943c5fbe6285e3000. The same commit introduces some changes on navigation.